.lgtm.yml

```
@@ -38,3 +38,13 @@ extraction:
         - libdconf-dev
         - socat
         - libdbus-1-dev
+        - e2fslibs-dev
+    after_prepare:
+      - git clone https://github.com/ostreedev/ostree.git ./ostree
```

```suggestion
      - git clone --branch v2020.8 --depth 1 --no-tags https://github.com/ostreedev/ostree.git ./ostree
```
since it seems 3e289b19 is the 2020.8 release commit.

(As per https://github.com/flatpak/flatpak/pull/4025)